ui/logviewer/App.jsx

```diff
@@ -214,6 +225,30 @@ class App extends React.PureComponent {
     }
   };
 
+  logFormatter = (log) => {
+    if (/INFO - GECKO\(\d+\)/.test(log)) {
+      // Do nothing
+    } else if (/INFO - TEST-(OK)|(PASS)/.test(log))
+      return <span style={{ color: 'green' }}>{log}</span>;
+    else if (/INFO - TEST-UNEXPECTED-FAIL/.test(log))
+      return <span style={{ color: 'red' }}>{log}</span>;
+    else if (/INFO - TEST-START/.test(log))
+      return <span style={{ color: 'blue' }}>{log}</span>;
+    else if (/INFO - TEST-/.test(log))
+      return <span style={{ color: 'yellow' }}>{log}</span>;
+    else if (/!!!/.test(log))
+      return <span style={{ color: [255, 0, 255] }}>{log}</span>;
+    else if (/Browser Chrome Test Summary$/.test(log))
+      return <span style={{ color: [255, 255, 255] }}>{log}</span>;
+    else if (/((INFO -)|([\s]+))(Passed|Failed|Todo):/.test(log))
+      return <span style={{ color: [255, 255, 255] }}>{log}</span>;
+    else if (/INFO/.test(log)) {
+      return <span style={{ color: [91, 127, 127] }}>{log}</span>;
+    } else {
+      return <span style={{ color: [91, 127, 127] }}>{log}</span>;
```

Comment:
I don't think specifying rgb colors in an array like this will work. Using a random color (such as 'green', 'orange', etc) is fine for now just to get this working.